core/src/main/java/org/fao/geonet/util/XslUtil.java

```diff
@@ -432,4 +437,89 @@ public static void setNoScript() {
     public static boolean allowScripting() {
         return allowScripting.get() == null || allowScripting.get();
     }
+    
+	public static String generateLineageSource(Object url) {
```

Comment:
Drop the method, it will be handled in the XSLT converting the WMC or OWS-C file.